pyiceberg/catalog/__init__.py

```diff
@@ -283,6 +292,20 @@ def delete_data_files(io: FileIO, manifests_to_delete: List[ManifestFile]) -> No
                 deleted_files[path] = True
 
 
+def _import_catalog(catalog_impl: str, properties: Properties) -> Optional[Catalog]:
+    try:
+        path_parts = catalog_impl.split(".")
+        if len(path_parts) < 2:
+            raise ValueError(f"py-catalog-impl should be full path (module.CustomCatalog), got: {catalog_impl}")
+        module_name, class_name = ".".join(path_parts[:-1]), path_parts[-1]
+        module = importlib.import_module(module_name)
+        class_ = getattr(module, class_name)
+        return class_(properties)
```

Comment:
```suggestion
        return class_(name, **properties)
```
The Catalog has one required argument "name", and take properties as kwargs. We need to pass the arguments like this otherwise the catalog implementation cannot pick-up the properties. The name can be obtained from `load_catalog`

Reply:
thanks, missed this when copying from `_import_file_io`, fixed.